installing/installing_openstack/installing-openstack-three-node.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-openstack-three-node"]
```

Comment:
I see that this module is a almost identical to the [GCP module](https://80595--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-three-node); maybe they could be merged and ifdef/ifndef statements could be used to consolidate two modules into one. This is not mandatory requirement.